app/src/main/java/org/simple/clinic/drugs/search/DrugSearchListItem.kt

```diff
@@ -21,9 +22,9 @@ sealed class DrugSearchListItem : PagingItemAdapter.Item<Event> {
 
   companion object {
 
-    fun from(searchResults: PagingData<Drug>, searchQuery: String): PagingData<DrugSearchListItem> {
+    fun from(searchResults: PagingData<Drug>, searchQuery: String, drugFrequencyToFrequencyChoiceItemMap: Map<DrugFrequency?, DrugFrequencyChoiceItem>): PagingData<DrugSearchListItem> {
```

Comment:
Can we put the arguments in separate lines, please? It crosses our character limit in a single line.